ci/promote-overlay/promote-overlay.sh

```diff
@@ -75,7 +75,7 @@ fi
 UPDATE_BRANCH_NAME="integration-service-${TARGET_OVERLAY}-update-"$(date '+%Y_%m_%d__%H_%M_%S')
 
 # GitHub repository details
-owner="redhat-appstudio"
+owner="konflux-ci"
```

Comment:
This still relates to infra-deployments, konflix-ci/infra-deployments doesn't exist

Reply:
We may update this `owner` in another PR after infra-deployments and integratio-service are migrated to `konflux-ci`.

Reply:
moved related changes to a different [pr](https://github.com/konflux-ci/integration-service-utils/pull/5)